articles/connections/database/custom-db/templates/index.md

```diff
@@ -0,0 +1,23 @@
+---
+description: Custom DB script templates
+topics:
+    - connections
+    - custom-database
+contentType: index
+useCase:
+    - customize-connections
+---
+# Custom Database Script Templates
+
+Auth0 provides the following custom database change password script templates that you can use when implementing password changing functionality for your users.
+
+While Auth0 has populated default templates in the Dashboard script editor, you can use the following links to recover the original code and notes once you've made and saved edits.
+
+::: next-steps
```

Comment:
When there are over 4-5 items this element looks bad I think. Consider replacing it with bullets https://auth0-docs-content-pr-6406.herokuapp.com/docs/connections/database/custom-db/templates